openwisp-config/Makefile

```diff
@@ -100,16 +100,11 @@ define Package/openwisp-config/postinst
 if [ ! -L $${IPKG_INSTROOT}/usr/sbin/openwisp_config ]; then
 	ln -s /usr/sbin/openwisp-config $${IPKG_INSTROOT}/usr/sbin/openwisp_config
 fi
```

Comment:
Was this intentionally not removed?

Reply:
The executable is not the problem, the initscript existing with two different names is the problem.